controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -256,7 +217,8 @@ public void deleteScope(final String scopeName, final SecurityContext securityCo
         long traceId = LoggerHelpers.traceEnter(log, "deleteScope");
 
         try {
-            authenticateAuthorize(scopeName, READ_UPDATE);
+            restAuthHelper.authenticateAuthorize(getAuthorizationHeader(),
+                    AuthResourceRepresentation.ofScope(scopeName), READ_UPDATE);
```

Comment:
I thought that we agreed that deleting a scope would be considered an operation on the parent (`/`).   The goal is for scopes to be provisioned by the administrator; the scope user shouldn't be able to delete it.    Therefore the solutions are:

```
restAuthHelper.authenticateAuthorize(getAuthorizationHeader(),
                    AuthResourceRepresentation.ofScopes(), READ_UPDATE);
```

or introduce a new verb:
```
restAuthHelper.authenticateAuthorize(getAuthorizationHeader(),
                    AuthResourceRepresentation.ofScope(scopeName), DELETE);
```

Reply:
If we add a new `DELETE` permission, then we should at least have a `CREATE` too. I'd rather not introduce only one. If we are to change the set of permissions we can grant, I'd rather rethink the model entirely and propose a new design. For this PR, my preference is that we stick to `READ` and `READ_UPDATE`, and accept its limitations.

Please check the discussion in the issue from this comment onward:

https://github.com/pravega/pravega/issues/3277#issuecomment-466900097